ios/FluentUI.Demo/FluentUIDemoTests/HUDTest_SwiftUI.swift

```diff
@@ -11,10 +11,55 @@ class HUDTestSwiftUI: BaseTest {
     override func setUpWithError() throws {
         try super.setUpWithError()
         app.staticTexts["SwiftUI Demo"].tap()
+        app.buttons["Present HUD for 3 seconds"].tap()
+        // sleeps for 5 seconds to wait for presented HUD to disappear
+        sleep(5)
     }
 
     // launch test that ensures the demo app does not crash and is on the correct control page
     func testLaunch() throws {
         XCTAssertTrue(app.navigationBars.element(matching: NSPredicate(format: "identifier CONTAINS %@", controlName)).exists)
     }
+
+    func testPresentation() throws {
+        let blocksInteractionSwitch: XCUIElement = app.switches["Blocks interaction"]
+        let presentHUDButton: XCUIElement = app.buttons["Present HUD for 3 seconds"]
+
+        presentHUDButton.tap()
+        // attempts to interact with Blocks interaction switch
+        blocksInteractionSwitch.tap()
+        // switch value should remain 1
+        XCTAssertTrue(blocksInteractionSwitch.value as? String == "1")
```

Comment:
I was thinking you could declare letBlocksInteractionSwitchInitialValue - blocksInteractionSwitch.value at the top of this test block, then you could just check whether or not the new values are equal to it. Just thought it might be cleaner but the comments you added also help!